server/src/entities/test.ts

```diff
@@ -0,0 +1,7 @@
+export type TestResults = {
+    patientId: number;
+    testResult: string;
+    typeOfTest: string;
+    dateOfTest: Date;
```

Comment:
Same thing here for the consistent naming

```suggestion
    testDate: Date;
```